src/components/App/NavBar.jsx

```diff
@@ -0,0 +1,68 @@
+import AppBar from "@material-ui/core/AppBar";
+import { NavLink } from "react-router-dom";
+import React from "react";
+import Toolbar from "@material-ui/core/Toolbar";
+import Typography from "@material-ui/core/Typography";
+import "./NavBar.scss";
+
+function NavBar() {
+  return (
+    <div>
+      <AppBar position="static">
+        <Toolbar className="navbar">
+          <div className="logo-content">
+            <Typography
+              style={{
+                fontSize: "1.7rem",
+                fontWeight: "700",
+                letterSpacing: "-0.08em",
+              }}
+            >
+              TeamUp
+            </Typography>
+          </div>
+          <div className="navbar-content">
+            <NavLink
+              activeStyle={{
+                color: "#64ffda",
+                fontWeight: "bold",
+              }}
+              className="navbar-label"
+              exact
+              style={{ color: "#ffffff", textDecoration: "none" }}
```

Comment:
In this case, activeStyle is a property of NavLink, the usual is to use “active” on scss but requires more changes to approach the same result. Another way I look out, is to put activeClassName="selected" and on scss .navbar-label.selected{ color: #64ffda !important; }, in this case in order to have the same result I used "!imporntant", since it was overriding, I’m not sure if applying that would be the best solution.

Reply:
NABD just keep the inline styles since those styles are not too much.